Release 0 11 (#463)
kestra-io/docs: content/docs/14.troubleshooting.md

          url: http://localhost:8080/
          variables:
            env-vars-prefix: "" # to avoid requiring KESTRA_ prefix on env vars
```

> ### Un-processable execution

Reviewer: ```suggestion
### Unprocessable execution
```

> Sometimes, things can go wrong in an unattended manner; for such extreme issues, we added a way to skip an execution that Kestra is not able to process.

Reviewer: ```suggestion
Sometimes, things can go wrong in an unattended manner; in such situations, you can skip an execution that Kestra is not able to process.
```
focuses on the reader

To do so, you can start the executor server (or the standalone server if not using a deployment with separate components) with a list of execution identifiers to skip.

```sh
kestra server executor --skip-executions 6FSPERUe1JwbYmMmdwRlgV,5iLGjTLOHAVGUGlsesFaMb
```

### Locked triggers

Flow schedule and polling triggers have locks to avoid concurrent trigger evaluation and concurrent execution of a flow for a trigger.

> Those can be seen on the **Administration -> Triggers** page. On this page, you can unlock a trigger if it is locked. Be careful that if it's not really locked, there is a risk or concurrent trigger evaluation or flow execution for this trigger.

Reviewer: ```suggestion
To see a list of triggers and inspect their current status, go to the **Administration -> Triggers** section in the Kestra UI. From here, you can unlock a trigger if it is locked. Keep in mind that there is a risk or concurrent trigger evaluation or flow execution for this trigger if you unlock it manually.
```
